go/logic/migrator.go

```diff
@@ -219,7 +219,11 @@ func (this *Migrator) onChangelogStateEvent(dmlEvent *binlog.BinlogDMLEvent) (er
 // listenOnPanicAbort aborts on abort request
 func (this *Migrator) listenOnPanicAbort() {
 	err := <-this.migrationContext.PanicAbort
+
+	//should cleanup before exit.
+	this.finalCleanup()
```

Comment:
I disagree. This is what panic is all about; getting here means something went wrong; attempting to clean up may make it worse.